snap/local/opt/kafka/start-wrapper.bash

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+"${SNAP}"/usr/bin/setpriv \
+    --clear-groups \
+    --reuid snap_daemon \
+    --regid snap_daemon -- \
+    "${SNAP}/bin/kafka-server-start.sh" "${SNAP_DATA}"/etc/kafka/server.properties
```

Comment:
**issue** the path does not look correct. Should then this be `$SNAP/opt/kafka/bin/kafka-server-start.sh`?

Reply:
**question** BTW...comparing this file with the other removed `start-kafka-wrapper.sh`, it seems that the logic is quite different and we do not include the log4j, jmx, server.properties AND broker.env. Is it because this would nevertheless do by the charm?

Also, In the start script, would it make sense to also provide the ability to add arguments when calling the bash script? Like using  `.... ${@}`?

Reply:
This is due to this change: 
```yaml
source: https://downloads.apache.org/kafka/${SNAPCRAFT_PROJECT_VERSION}/kafka_2.13-${SNAPCRAFT_PROJECT_VERSION}.tgz
```

Now, Kafka installs into the root filesystem (as one would expect with a ROCK), and as the `bin` commands are a 'part' of the Kafka tarball, they're in `bin`. It's what we're doing with ZK also. AFAIK, `opt/kafka` is for additional stuff, in this case our `*-wrapper.bash` and prometheus exporter.